src/services/Store/store/account.slice.ts

```diff
@@ -263,7 +265,7 @@ export const getMergedTxHistory = createSelector(
           return {
             ...tx,
             timestamp: tx.timestamp || 0,
-            txType: deriveTxType(accounts, tx) || ITxHistoryType.UNKNOWN,
+            txType: (deriveTxType(txTypeMetas, accounts, tx) || ITxHistoryType.UNKNOWN) as TxType,
```

Comment:
Why this type cast?

Reply:
We have 2 different txType types: one is an enum (for the native transaction types in the app), and the other is just a string (which comes from the api). For the purposes of txHistory, im just having it cast as one type so that we can reference it that way in the recent transactions components. I've moved this cast inside the function